src/components/optimizerTab/optimizerForm/TeammateCard.tsx

```diff
@@ -265,11 +271,14 @@ const TeammateCard = (props: { index: number }) => {
           </Flex>
           <Flex vertical gap={5}>
             <div style={{ width: `${rightPanelWidth}px`, height: `${rightPanelWidth}px`, borderRadius: '10px' }}>
-              <img
-                width={rightPanelWidth}
-                height={rightPanelWidth}
-                src={Assets.getCharacterAvatarById(teammateCharacterId)}
-              />
+              <a>
```

Comment:
In general I think <a> tags are not the right way to do this, seems like we just want the pointer cursor behavior so styling the image/div with

```
style={{
  cursor: 'pointer',
}}
```

is probably the best way. Same in the teammate code below and [OptimizerTabCharacterPanel.tsx](https://github.com/fribbels/hsr-optimizer/pull/579/files#diff-02a798201eb91a5421e6892589947adfc054890327ad66eb16a0757f7bfc5217)